### ISSUE TITLE
Broken compatibility with Archi 4.9.1

### ISSUE THREAD

**Issue by mlast31** [archilight/torchlight-azure#2]
Since release 4.9.1, the .zip file is not accepted anymore by Archi. Archi sends an error saying that it is not an Archi plug-in.

**Comment by tumesh** [comment]
@mlast31 I too have same issue, Alternatively you could unzip and copy underneath folder  i.e., com.torchlight.azure_release v0.1 to drop-in folder "User\AppData\Roaming\Archi4\dropins". 

**Installation of plugin**
1. Download the latest plugin from the releases area.
2. Unpack the zip file and copy the folder to the plugins directory of the Archi installation.

However, I still have issue in using those stencils or types. Here is the attached screen shot. Let me know if this works for you ?
![image](https://user-images.githubusercontent.com/18109627/142699382-35d986b6-7e8e-4194-a120-ba3d727f58f6.png)

**Comment by mlast31** [comment]
@tumesh Yes, I get the same error when I try to use one. And why I try to open an existing diagram, I get a list of errors ( I guess one by stencil:
![image](https://user-images.githubusercontent.com/94390586/142860118-81a8b295-3644-4c4a-a565-6ba10791c20d.png)

**Comment by tumesh** [comment]
> @tumesh Yes, I get the same error when I try to use one. And why I try to open an existing diagram, I get a list of errors ( I guess one by stencil: ![image](https://user-images.githubusercontent.com/94390586/142860118-81a8b295-3644-4c4a-a565-6ba10791c20d.png)

I am not sure if only diagrams that Azure stencils were added, I got similar exception for other Diagrams without Azure stencils, such as Metamodel View. I don't remember adding any new stencils to it.. Does it mean that the plug in added it ? 

I uninstalled plugin and it started working, I guess we will have to wait until the plug in is fully ready.

**Comment by mlast31** [comment]
Other diagrams without this plugin stencils work fine for me.

**Comment by adubu** [comment]
With the implementation of specializations in Archi 4.9, I guess that this plugin is redundant? 
Of course that doesn't really help, as I/we now have a bunch of diagrams that need converting (manually I guess) over to the new specialization functionality.

**Comment by riaadh** [comment]
Hello, 
I have the same issue.
Is this project dead? any alternatives? thanks
